docs/index.md

```diff
@@ -7,10 +7,8 @@ Welcome to the offical documentation of `disgames`! You can find our source code
 
 If you have any questions, check out our [faq](./faq.md)
 
-- [Disgames](#disgames)
-  - [Contents](#contents)
-  - [Installation](#installation)
-  - [Extra](#extra)
+- [Installation](#installation)
+- [Extra](#extra)
```

Comment:
This is to follow convention and link to **all** anchor tags in the file. Revert this changes

Reply:
wait a sec

Reply:
ya ok, just remove the contents part... just have it like 
- [Disgames](#disgames)
    - [Installation](#installation)
    - [Extra](#extra)

Reply:
K